third_party/move/move-vm/transactional-tests/tests/module_publishing/republish_module_skip_compatible_linking_hack_struct.exp

```diff
@@ -1,30 +0,0 @@
-processed 5 tasks
-
-task 2 'publish'. lines 18-23:
-Error: Unable to publish module '0000000000000000000000000000000000000000000000000000000000000042::A'. Got VMError: {
```

Comment:
Why this is not an error anymore? Is this because we skipped checks in the test and tried to run?